### PR TITLE
Feature add team introduction photos

### PR DESCRIPTION
**Pull request by maciejSamerdak**
Replaces text boxes with graphics and additionally fixes responsiveness issues with top description.

### CODE REVIEW

**Comment by kbeker** [comment]
@maciejSamerdak I can't find these graphics yet in google drive. Did you already uploaded them or where they are?

File: company_website/templates/team_introduction_page_partials/company_introduction.haml [hunk 1]
```diff
@@ -30,49 +30,22 @@
...
         .right-container.right-column
-            .outer-text-div.outer-understand-text-div{:data-aos => "fade-up"}
-                .inner-text-div.inner-understand-text-div
-                    .topic-row
-                        .topic UNDERSTAND
-                    .text-row
-                        %p.description.understand-part-description
-                            We dig deep to understand your business, goals and requirements.
-                            We need to create partnership and be understanding to each other in order to provide the best solutions.
+            %img.lazyload.image_16{:data-src => "{% static 'team_introduction_page/images/image_16.jpg' %}", :alt => "TEMP", :title => "TEMP", :data-aos => "fade-up"}
```
Comment (kbeker) [inline company_website/templates/team_introduction_page_partials/company_introduction.haml:39]: Images should have different sizes for different screen resolutions

File: company_website/templates/team_introduction_page_partials/company_introduction.haml [hunk 2]
```diff
@@ -30,49 +30,22 @@
...
-            .outer-text-div.improve-text-div{:data-aos => "fade-up"}
-                .inner-text-div.inner-improve-text-div
-                    .topic-row
-                        .topic IMPROVE
-                    .text-row
-                        .description.improve-part-description
-                            Our brains hate being idle.
-                            Passionate about our work, we are constantly moving forward, innovating and improving.
-                            We think that it is the key to success.
+            %img.lazyload.image_17{:data-src => "{% static 'team_introduction_page/images/image_17.jpg' %}", :alt => "TEMP", :title => "TEMP", :data-aos => "fade-up"}
```
Comment (kbeker) [inline company_website/templates/team_introduction_page_partials/company_introduction.haml:48]: Images should have different sizes for different screen resolutions

File: company_website/templates/team_introduction_page_partials/company_introduction.haml [hunk 3]
```diff
@@ -82,32 +55,14 @@
...
-            .outer-text-div.be-honest-text-div{:data-aos => "fade-up"}
-                .inner-text-div.inner-be-honest-text-div
-                    .topic-row
-                        .topic
-                            BE HONEST
-                    .text-row
-                        %p.description.be-honest-part-description
-                            Transparency and good communication are essential in achieving the best results and solving problems.
-                            We are hones with you and we expect you to be honest with us.
+            %img.lazyload.image_18{:data-src => "{% static 'team_introduction_page/images/image_18.jpg' %}", :alt => "TEMP", :title => "TEMP", :data-aos => "fade-up"}
```
Comment (kbeker) [inline company_website/templates/team_introduction_page_partials/company_introduction.haml:58]: Images should have different sizes for different screen resolutions

File: company_website/templates/team_introduction_page_partials/company_introduction.haml [hunk 4]
```diff
@@ -118,11 +73,4 @@
...
         .right-container.right-column
-            .outer-text-div.self-organized-2-text-div{:data-aos => "fade-up"}
-                .inner-text-div.inner-self-organized-part-2-text-div
-                    .topic-row
-                        .topic
-                            FEEL GOOD
-                    .text-row
-                        %p.description.self-organized-part-2-description
-                            Maybe we don't save the world while working on the projects but we always remember that at the same time we can fell good, have fun, make friends and drink good coffee.
+            %img.lazyload.image_19{:data-src => "{% static 'team_introduction_page/images/image_19.jpg' %}", :alt => "TEMP", :title => "TEMP", :data-aos => "fade-up"}
```
Comment (kbeker) [inline company_website/templates/team_introduction_page_partials/company_introduction.haml:76]: Images should have different sizes for different screen resolutions

**Comment by maciejSamerdak** [comment]
You can find all the images here, along with the already existing ones:
https://drive.google.com/drive/folders/1WwyfPnrQLDrsud__pTXkrJmW-QIGeMbP

**Pull request merged**